vispy/__init__.py

```diff
@@ -27,6 +27,13 @@
 # Nice string for the version (mimic how IPython composes its version str)
 __version__ = '-'.join(map(str, version_info)).replace('-', '.', 2).strip('-')
 
+from argparse import ArgumentParser
+
+parser = ArgumentParser()
+parser.add_argument('--vispy-glir-file', help='Export glir commands to specified file.')
+args = parser.parse_args()
+glir_file = args.vispy_glir_file
+
```

Comment:
Should we use the system in https://github.com/vispy/vispy/blob/5b01479e829e8f4b2dd348dff51ce864c1f834a5/vispy/util/config.py#L74 instead?